CONTRIBUTING.md

```diff
@@ -0,0 +1,856 @@
+# Bug reports
+
+Use the [issue tracker](https://github.com/lhorie/mithril.js/issues). Do check to make sure your bug hasn't already been filed. Please give the following information, where possible:
+
+1. The version of Mithril you're using, whether it's the dev version or [the version on npm](http://npm.im/mithril.js). The version on npm may not have all the latest bug fixes, so your bug might very well be fixed in the dev version.
```

Comment:
Yes, adding browser/version will also help in debugging issues

Personally, I prefer keeping the trailing commas ("your house, your rules"), it was just an example of an edge case where knowing if the minified version is being used would help with debugging. It's also possible that there be a regression in the minifier at some point in the future that would cause the two versions to behave differently.